### ISSUE TITLE
Mention the dependency on pandoc in the Readme

### ISSUE THREAD

**Issue by oantolin** [alphapapa/pocket-reader.el#26]
It's a little surprising to install this package and try to use it only to be told `pandoc` cannot be found. A sentence mentioning you need `pandoc` in the Readme or package commentary (ideally both), would be welcome.

**Comment by alphapapa** [comment]
Pandoc is not required to use this package.  The issue is that `pocket-reader-open-url-default-function` and `pocket-reader-pop-to-url-default-function` default to `org-web-tools-read-url-as-org`, which uses Pandoc.  It would probably be better to change those options' defaults to call Emacs's built-in functions instead.

**Comment by oantolin** [comment]
I see, thanks! I'll try it again, then.

**Comment by oantolin** [comment]
Sorry, I should have closed this long ago. I tried setting both `pocket-reader-open-url-default-function` and `pocket-reader-pop-to-url-default-function` to `#'eww` and so far so good.

**Comment by alphapapa** [comment]
Thanks.  I'll leave this open to consider changing the defaults.

**Comment by oantolin** [comment]
It seems to me that this package does not require `org-web-tools` other than for the default value of `pocket-reader-open-url-default-function` and `pocket-reader-pop-to-url-default-function`. I'd suggest:

- Removing the dependency on `org-web-tools`.
- Making the default for those functions `eww`, because it comes with Emacs.
- Changing the `defcustom` declaration for those variables so it "recommends" org-web-tools:

```elisp
(defcustom pocket-reader-open-url-default-function
  #'eww
  "Default function to open items."
  :type '(choice
          (const :tag "Use eww" eww)
          (const :tag "Use org-web-tools" org-web-tools-read-url-as-org)
          function))
```

I also don't think this is big deal at all, so feel free to ignore this suggestion (and to close this issue as well).

**Comment by pataquets** [comment]
Anyway, mentioning in docs that `pandoc`/`org-web-tools` enables better web handling would be worth mentioning. Otherwise, the feature might go unnoticed.

**Comment by alphapapa** [comment]
As much as I'd like to promote the use of `org-web-tools`, I'm leaning towards Omar's suggestion of defaulting to using `eww` and removing the dependency.  I doubt many people actually want to run the pages through Pandoc and read the output as Org, anyway, and EWW's better than it used to be.

**Comment by pataquets** [comment]
Fair enough. Honestly, I can't come up with any use case which might benefit, I was just assuming that `org-web-tools` handling was superior. But if EWW is better nowadays and `org-web-tools` can't bring any substantial improvement, keeping dependencies lean makes the most sense. It would be useful only as a `pocket-reader-open-url-default-function` usage example, at most.
Thanks for considering and for the information.
